src/com/stuypulse/stuylib/math/Vector2D.java

```diff
@@ -24,7 +24,10 @@ public Vector2D() {
      * Initialize Vector2D using an array of 2 doubles
      * @param in an array of 2 numbers
      */
-    public Vector2D(double[] in) {
+    public Vector2D(double[] in) throws Exception {
```

Comment:
Don't use a checked exception here. Use RuntimeException instead. I personally hate checked exceptions, but this isn't the kind of error that the user will know how to fix.

Reply:
I will say, I don't think this constructor needs to exist.

Reply:
this constructor is based off of the way some api's represent vector 2d